features/schools/onboarding/school_profile.feature

```diff
@@ -54,7 +54,7 @@ Feature: School Profile
       | Subjects                                   | Maths                                                                                                                                                                                                                                                       |
       | Description                                | We have a race track                                                                                                                                                                                                                                        |
       | School experience details                  | A really good one                                                                                                                                                                                                                                           |
-      | Teacher training links                     | Yes - We run our own training. Teacher training information                                                                                                                                                                                                 |
+      | Teacher training links                     | Yes - We run our own training. Find out more about our teacher training.                                                                                                                                                                                                 |
```

Comment:
I removed the full stop from the link text in a previous part of the PR. Can we remove the full stop here?